README.adoc

```diff
@@ -0,0 +1,131 @@
+Official https://github.com/joelittlejohn/jsonschema2pojo[jsonschema2pojo] Gradle plugin
+========================================================================================
+:toc:
+:toc-placement: preamble
+:toclevels: 1
+:showtitle:
+
+// Need some preamble to get TOC:
+{empty}
+
+== Introduction
+This plugin is aiming to take raw JSON or YAML raw files or schemas and convert to Java or Scala POJOs (Plain Old Java Object).
+
+Currently it suppots features like generating using Jackson, GSON and Mocha1 annotations and JSR305.
+
+Please note, that JSR305 is purely supported, and present only if you don't want to use a proper JSON validation for some reasons.
+
+== Adding a plugin to your build.gradle
+
+[source,gradle]
+----
+plugins {
+  id "org.jsonschema2dataclass" version "2.1.0" // Please update version as new versions are out
+}
+----
+
+Please refer https://plugins.gradle.org/plugin/org.jsonschema2dataclass[Gradle Plugins page] for further details.
+
+== Usage Example
+
+Usage for Android and for Java is basically the same. You need to declare plugin usage and configure plugin parameters if needed.
+
+To execute just a java classes generation you need to run `generateJsonSchema2DataClass` gradle task. Otherwise, `compileJava` and appropriate android tasks are already dependent to this plugin.
+
+To start using a plugin you need follow this recipe:
+
+.Minimal java example how to use the plugin
+[source,gradle]
+----
+plugins{
+    id "org.jsonschema2dataclass" version "2.1.0"
+}
+
+jsonschema2pojo {
+    targetPackage = 'org.example.api' // specify package for your needs
+    // ... — add other supported options as you need
+}
+----
+
+Additionally you can copy one of sample Java and Android projects you can find in https://github.com/jsonschema2dataclass/js2d-gradle/tree/master/demo[demo] folder
+
+
+== Accepted parameters documentation
+<details>
+<summary>Accepted parameters</summary>
+
+[NOTE]
+📝 Documentation is a bit outdated on the jsonschema2pojo pages.
+Help wanted for this task
+
+[CAUTION]
+⚠️ Formatting is broken in some cells, I need help to fix it up
```

Comment:
This indicates the error

![image](https://user-images.githubusercontent.com/3314607/111077200-eccf7080-84ef-11eb-9471-3232e171ea59.png)

![image](https://user-images.githubusercontent.com/3314607/111077207-f6f16f00-84ef-11eb-8a2f-c76832376bd9.png)

Reply:
Maybe it helps you: you can easy check current state 

![image](https://user-images.githubusercontent.com/3314607/111077267-4768cc80-84f0-11eb-9f9a-10822f9e0341.png)

Reply:
Thanks, I hoped, that these tags are supported for `asciidoc` as well. Apparently they aren't